pkg/image/prune/prune.go

```diff
@@ -900,32 +900,32 @@ func pruneManifests(
 
 // imageDeleter removes an image from OpenShift.
 type imageDeleter struct {
-	images client.ImageInterface
+	images imageclient.ImagesGetter
 }
 
 var _ ImageDeleter = &imageDeleter{}
 
 // NewImageDeleter creates a new imageDeleter.
-func NewImageDeleter(images client.ImageInterface) ImageDeleter {
+func NewImageDeleter(images imageclient.ImagesGetter) ImageDeleter {
 	return &imageDeleter{
 		images: images,
 	}
 }
 
 func (p *imageDeleter) DeleteImage(image *imageapi.Image) error {
 	glog.V(4).Infof("Deleting image %q", image.Name)
-	return p.images.Delete(image.Name)
+	return p.images.Images().Delete(image.Name, metav1.NewDeleteOptions(0))
```

Comment:
old code was this aggresive with the delete options?

Reply:
Hmmm... there was no such option in the old code, and since this is pruning I assumed this will be the best option.

Reply:
> Hmmm... there was no such option in the old code, and since this is pruning I assumed this will be the best option.

pass nil or empty object.  Pretty sure this one takes a nil.